packages/client/src/reducers/api.ts

```diff
@@ -55,7 +55,11 @@ export const api = createApi({
       transformResponse: (response: { service_id: string }) => response.service_id,
     }),
     getUser: builder.query<User, void>({
-      query: () => '/proxy/auth/user',
+      query: () => ({
+        url: '/users/user',
+        method: 'POST',
+        body: { theme: localStorage.getItem(THEME_LS_KEY) ? localStorage.getItem(THEME_LS_KEY) : 'light' },
```

Comment:
Можно избежать дублирования
```suggestion
        body: { theme: localStorage.getItem(THEME_LS_KEY) ?? 'light' },
```